samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/TotalScoresBean.java

```diff
@@ -220,6 +232,53 @@ protected void init() {
 		
 		agents = newAgents;	
 	}
+
+  public boolean isOneSelectionType() {
+    if (this.getPublishedAssessment() != null) {
+      for (Object itemObject : this.getPublishedAssessment().getDefaultSection().getItemArray()) {
```

Comment:
This only iterates over the main section, am I right? What happens with other sections?

Reply:
I didn't take in mind. Sorry, I will fix it now